jest/private/v28.1.0/defs.bzl

```diff
@@ -4,10 +4,10 @@
 load("@aspect_rules_js//npm/private:npm_linked_packages.bzl", "npm_linked_packages")
 load("@npm__at_ampproject_remapping__2.2.0__links//:defs.bzl", link_0_store = "npm_link_imported_package_store")
 load("@npm__at_babel_code-frame__7.16.7__links//:defs.bzl", link_1_store = "npm_link_imported_package_store")
-load("@npm__at_babel_compat-data__7.17.10__links//:defs.bzl", link_2_store = "npm_link_imported_package_store")
-load("@npm__at_babel_core__7.18.2__links//:defs.bzl", link_3_store = "npm_link_imported_package_store")
+load("@npm__at_babel_compat-data__7.18.5__links//:defs.bzl", link_2_store = "npm_link_imported_package_store")
```

Comment:
it's too bad the delta got so big -looks like re-mirroring the library wasn't even required?